src/using-document-creator.mdx

```diff
@@ -0,0 +1,179 @@
+---
+name: Using Document Creator
+route: /using-document-creator
+---
+
+# Using TradeTrust Document Creator
+
+You can use the [document creator](https://creator.tradetrust.io) to create your document.
+
+To use this functionality, you will need a configuration file(`.json` file) and some pre-requisite informations.
+
+## What is document creator about?
+
+It's to provide a UI to the staffs that are involve with the daily operations who do not need to know anything about blockchain.
+
+## Why do we do this?
+
+Mainly because we want to cater to small firms or companies that doesn't really have the capabilities to build deep integration with their existing softwares or workflows.
+
+## How is this document creator helping small firms and companies?
+
+One way is for the IT department or someone who is tech savvy enough, can set up the configuration file and clerks or staffs can use this document creator to create the documents.
+
+## Pre-requisite
+
+Before you construct your configuration file(`.json` file), please ensure you have these:
```

Comment:
there are 3 TODOs page in the list :eyes:

Reply:
yup, which will be updated soon, in other PRs